lib/asset/transferpart.js

```diff
@@ -46,7 +47,8 @@ class TransferPart {
             transferAsset,
             targetUrls,
             contentRange,
-            headers
+            sourceHeaders,
+            targetHeaders
```

Comment:
This shouldn't be necessary, since the source and target `asset` already have headers associated with them: https://github.com/adobe/node-httptransfer/blob/master/lib/asset/asset.js#L67-L74

The reason target headers are here as well is because the original target asset is pointing at AEM, and after initiate upload the target is pointing at another service (Azure) in this case which may need its own headers.

This isn't the case for sources, since HTTP supports range requests already so the source asset is the same URL as the URL as in the part. I do think this behavior is difficult to extract though. So I'd leave `targetHeaders` here to make it clear that those are target headers.

And then the `sourceHeaders` getter below, I'd change to return `this[PRIVATE].transferAsset.source.headers` for convenience.